vng/base/views.py

```diff
@@ -95,31 +110,158 @@ def vngp1_predict_license_plate(request):
         company.company_ratings = license_plate_company_data['company_ratings']
         company.latitude = lat
         company.longitude = lng
+        
+        try:
+            pnt_company = Point(lat, lng)
+            print(pnt_company)
+            company.geom = pnt_company
+        except ValueError:
+            return Response({
+                "status": False,
+                "errMsg": "No GPS Data Found! Please Upload Another Image."
+            })
         company.save()
+        image_id = uuid.uuid4()
         targetImage = TargetImage(company=company)
-        targetImage.image = file
+        targetImage.image_id = image_id
+        targetImage.image = File(file=open("image.jpg", 'rb'), name=image_name)
+        targetImage.image_name = image_name
+        try:
+            pnt_image = Point(lat, lng)
+            targetImage.geom = pnt_image
+        except ValueError:
+            return Response({
+                "status": False,
+                "errMsg": "No GPS Data Found! Please Upload Another Image."
+            })
         targetImage.save()
-        os.remove("image.png")
-        
-        for license_number in license_plate_company_data['license_number']:
-            licensePlate = LicensePlate(company=company, target_image=targetImage)
-            licensePlate.license_number = license_number
-            licensePlate.save()
+        get_image = TargetImage.objects.get(image_id=image_id)
+        updateCompanyImageUrl = Company.objects.get(company_id=id)
+        updateCompanyImageUrl.image_url = get_image.image.url
+        updateCompanyImageUrl.save()
+        if license_plate_company_data['license_number'] != "":
+            for license_number in license_plate_company_data['license_number']:
+                license_id = uuid.uuid4()
+                licensePlate = LicensePlate(company=company, target_image=targetImage)
+                licensePlate.license_number = license_number
+                licensePlate.license_plate_id = license_id
+                try:
+                    pnt_license = Point(lat, lng)
```

Comment:
I'm pretty sure it should be longitude, latitude

Reply:
it is resolved.